app/src/main/java/se/inera/intyg/minaintyg/certificate/service/GetCertificateService.java

```diff
@@ -0,0 +1,39 @@
+package se.inera.intyg.minaintyg.certificate.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import se.inera.intyg.minaintyg.certificate.service.dto.GetCertificateRequest;
+import se.inera.intyg.minaintyg.certificate.service.dto.GetCertificateResponse;
+import se.inera.intyg.minaintyg.integration.api.certificate.GetCertificateIntegrationRequest;
+import se.inera.intyg.minaintyg.integration.api.certificate.GetCertificateIntegrationService;
+import se.inera.intyg.minaintyg.logging.MonitoringLogService;
+import se.inera.intyg.minaintyg.user.UserService;
+
+@Service
+@RequiredArgsConstructor
+public class GetCertificateService {
+
+  private final GetCertificateIntegrationService getCertificateIntegrationService;
+  private final MonitoringLogService monitoringLogService;
+  private final UserService userService;
+  private final FormattedCertificateConverter formattedCertificateConverter;
+
+  public GetCertificateResponse get(
```

Comment:
Silly comments, but feel free to put the arguments on same line and remove the empty line on row 37 (after the return statement).